loadgen/contract/agent-prepare-loadgen.js

```diff
@@ -300,7 +300,9 @@ export default async function startAgent({
   // Use `when` as older versions of agoric-sdk cannot accept a promise
   // See https://github.com/Agoric/agoric-sdk/issues/3837
   /** @type {ERef<import('../types.js').AttenuatedAMM>} */
-  const amm = E.when(withFee(ammInstance), E(zoe).getPublicFacet);
+  const amm = E.when(withFee(ammInstance), (instance) =>
```

Comment:
The parens around the single-identifier-parameter surprises me. Are we not linting this repository?

Reply:
```suggestion
  const amm = E.when(withFee(ammInstance), instance =>
```

Reply:
The prettier rules in this repo are different (I updated to prettier 2 much before agoric-sdk did, and kept with the defaults)

Reply:
Please file a bug to fix the prettier rules on this repository (and others?) to remove these parens. Thanks.

Reply:
Filed: https://github.com/Agoric/testnet-load-generator/issues/94